all_protos/ibc/lightclients/tendermint/v1/tendermint.proto

```
@@ -4,8 +4,8 @@ package ibc.lightclients.tendermint.v1;
 
 option go_package = "github.com/cosmos/ibc-go/v8/modules/light-clients/07-tendermint;tendermint";
 
-import "tendermint/types/validator.proto";
-import "tendermint/types/types.proto";
+import "cometbft/types/v1/validator.proto";
```

Buf (1.54.0)

import "cometbft/types/v1/validator.proto": file does not exist

(COMPILE)